examples/passport-oauth2-login/src/authentication-strategies/strategy-adapter.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/authentication-passport
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {UserProfileFactory} from '@loopback/authentication';
```

Comment:
I think we should leverage the adapter from https://github.com/strongloop/loopback-next/tree/master/extensions/authentication-passport

Reply:
agreed, symlinks to master packages enabled now.